media-gfx/openscad/openscad-2021.01.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit elisp-common qmake-utils xdg
+
+SITEFILE="50${PN}-gentoo.el"
+
+DESCRIPTION="The Programmers Solid 3D CAD Modeller"
+HOMEPAGE="https://www.openscad.org/"
+SRC_URI="https://github.com/${PN}/${PN}/releases/download/${P}/${P}.src.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="ccache emacs"
+RESTRICT="test"
```

Comment:
Why?

Reply:
They are not ready yet. Most of them work, if you build them with cmake, which isn't yet officially supported. See https://github.com/gentoo/gentoo/pull/19412/files#diff-245dbc481dcedeb832abdc08edc62a2277fa08d9f234768a3ce8e0f0d4a2a187

Reply:
Can you add a comment to this effect in the ebuild?